testing/adapter/index.js

```diff
@@ -198,10 +199,54 @@ async function nearToEthUnlock ({
   exit(0)
 }
 
+// View
+async function getBridgeOnNearBalance ({
+  nearReceiverAccount,
+  nearMasterAccount: nearMasterAccountId,
```

Comment:
Why using different name here?
```suggestion
  nearMasterAccount,
```